core/ledger/kvledger/txmgmt/statedb/statecouchdb/commit_handling_test.go

```diff
@@ -150,3 +150,128 @@ func TestBuildCommitters(t *testing.T) {
 	committer, err = db.buildCommitters(badBatch)
 	assert.EqualError(t, err, "nil version not supported")
 }
+
+func TestExecuteCommitter(t *testing.T) {
+	env := testEnv
+	env.init(t, &statedb.Cache{})
+	defer env.cleanup()
+
+	versionedDB, err := testEnv.DBProvider.GetDBHandle("test-execute-committer")
+	assert.NoError(t, err)
+	db := versionedDB.(*VersionedDB)
+
+	committerDB, err := db.getNamespaceDBHandle("ns")
+	assert.NoError(t, err)
+	couchDocKey1, err := keyValToCouchDoc(&keyValue{
+		key:            "key1",
+		revision:       "",
+		VersionedValue: &statedb.VersionedValue{Value: nil, Metadata: nil, Version: version.NewHeight(1, 1)},
+	})
+	assert.NoError(t, err)
+	couchDocKey2, err := keyValToCouchDoc(&keyValue{
+		key:            "key2",
+		revision:       "",
+		VersionedValue: &statedb.VersionedValue{Value: nil, Metadata: nil, Version: version.NewHeight(1, 1)},
+	})
+	assert.NoError(t, err)
+
+	committers := []*committer{
+		{
+			db:             committerDB,
+			batchUpdateMap: map[string]*batchableDocument{"key1": {CouchDoc: *couchDocKey1}},
+			namespace:      "ns",
+			cacheKVs:       make(statedb.CacheKVs),
+			cacheEnabled:   true,
+		},
+		{
+			db:             committerDB,
+			batchUpdateMap: map[string]*batchableDocument{"key2": {CouchDoc: *couchDocKey2}},
+			namespace:      "ns",
+			cacheKVs:       make(statedb.CacheKVs),
+			cacheEnabled:   true,
+		},
+	}
+
+	err = db.executeCommitter(committers)
+	assert.NoError(t, err)
+
```

Comment:
can you add a check for key/values finally committed in the db

Reply:
@manish-sethi check has been added, thanks.